mobileApp/jobs/edxAppAndroidPr.groovy

```diff
@@ -0,0 +1,65 @@
+package mobileApp
+
+import org.yaml.snakeyaml.Yaml
+import static org.edx.jenkins.dsl.JenkinsPublicConstants.JENKINS_PUBLIC_LOG_ROTATOR
+
+/* stdout logger */
+Map config = [:]
+Binding bindings = getBinding()
+config.putAll(bindings.getVariables())
+PrintStream out = config['out']
+
+/* Map to hold the k:v pairs parsed from the secret file */
+Map ghprbMap = [:]
+try {
+    out.println('Parsing secret YAML file')
+    String ghprbConfigContents = new File("${GHPRB_SECRET}").text
+    Yaml yaml = new Yaml()
+    ghprbMap = yaml.load(ghprbConfigContents)
+    out.println('Successfully parsed secret YAML file')
+}
+catch (any) {
+    out.println('Jenkins DSL: Error parsing secret YAML file')
+    out.println('Exiting with error code 1')
+    return 1
+}
+
+pipelineJob('edx-app-android-build') {
+    
+    description('Build the edX Android app and archive it on Jenkins')
+
+    definition {
+
+        logRotator JENKINS_PUBLIC_LOG_ROTATOR(7)
+
+        triggers {
+            githubPullRequest {
+                admins(ghprbMap['admin'])
+                useGitHubHooks(true)
+                triggerPhrase('jenkins run tests')
+                onlyTriggerPhrase(false)
+                userWhitelist(ghprbMap['userWhiteList'])
+                orgWhitelist(ghprbMap['orgWhiteList'])
+                extensions {
+                    commitStatus {
+                        context('jenkins/build')
+                    }
+                }
+            }
+        }
+
+        cpsScm {
+            scm {
+                git {
+                    remote {
+                        credentials('jenkins-worker')
+                        github('edx/edx-app-android', 'https', 'github.com')
+                        refspec('+refs/heads/*:refs/remotes/origin/* +refs/pull/*:refs/remotes/origin/pr/*')
```

Comment:
Do we need this broad of a refspec?  Seems like we could get away with just the current PR branch, like https://github.com/edx/jenkins-job-dsl/blob/master/platform/jobs/pipelines/edxPlatformPipelinePr.groovy#L116

Reply:
done